### PR TITLE
Group documents in docs-internal

### PR DESCRIPTION
**Pull request by 0x6431346e**
# 🦅 Pull Request

Trying to organize a bit the various documents we have in order to get a clearer big picture.

## ✔️ PR Todo

- [ ] Included links to related issues/PRs
- [ ] Added/updated unit tests for this change
- [ ] Added/updated the necessary documentation
- [x] Cleared dependencies on other modules that have to be released before merging this

### CODE REVIEW

File: docs-internal/Labels.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+# Labels
```
Comment (sohkai) [inline docs-internal/Labels.md]: ❤️ This is really nice! I should add something similar for aragon/aragon 😄

Comment (0x6431346e) [reply docs-internal/Labels.md]: Perhaps we can have something like `aragon/community-standards` where anything non-project specific will go there? Like this, `CONTRIBUTING.md`, eslint configs, etc.

Comment (sohkai) [reply docs-internal/Labels.md]: Yeah, we've been thinking of exactly something like this. Kind of like a "Aragon github standards" that has templates for READMEs, release notes, etc.

File: docs-internal/README.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+Current state of things:
+
+- [../docs/](/docs/Intro.md) - the API interface
+- [../CONTRIBUTING](/CONTRIBUTING.md) - contributing guidelines
+- [../README.md](/README.md) - ??
+- [../packages/aragon-cli/README.md](/packages/aragon-cli/README.md) - ??
+- [../packages/create-aragon-app/README.md](/packages/create-aragon-app/README.md) - ??
+- [Specification](/docs-internal/Specification.md) - the technical architecture and implementation details
+- [Testing](/docs-internal/Testing.md) - the testing approach
+- [Labels](/docs-internal/Testing.md) - the issue labeling system
```
Comment (0xGabi) [inline docs-internal/README.md]:
```suggestion
[Labels](/docs-internal/Labels.md) - the issue labeling system
```

File: docs-internal/Specification.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
+# Specification
```
Comment (0xGabi) [inline docs-internal/Specification.md]: Brilliant, thanks to start shaping the ideas ❤️

File: docs-internal/next/Specification.md [hunk 4]
```diff
@@ -1,4 +1,25 @@
-# Specification
+# Specification for the next versions of aragonCLI
```
Comment (0xGabi) [inline docs-internal/next/Specification.md]: Brilliant, thanks to start shaping the ideas ❤️

File: docs-internal/README.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
+# Internal documents regarding the project development & maintenance
+
+Current state of things:
+
+- [../docs/](/docs/Intro.md) - the API interface
+- [../CONTRIBUTING](/CONTRIBUTING.md) - contributing guidelines
+- [../README.md](/README.md) - ??
```
Comment (0xGabi) [inline docs-internal/README.md]: Maybe not include a description for README files as they are self explained?

**Pull request merged**